docs/config.md

```diff
@@ -1397,6 +1398,43 @@ Default: `auto` (DHT + IPNI)
 
 Type: `optionalString` (`null`/missing means the default)
 
+
+### `Routing.AcceleratedDHTClient`
+
+Utilizes an alternative DHT client that searches for and maintains more information
+about the network in exchange for being more performant.
+
+This is not compatible with `Routing.Type` `custom`. If you are using composable routers
+you can configure this individualy on each router.
+
+When it is enabled:
+- DHT operations should complete much faster than with it disabled
+- A batching reprovider system will be enabled which takes advantage of some
+  properties of the experimental client to very efficiently put provider records into the network
+- The standard DHT client (and server if enabled) are run alongside the accelerated DHT client
```

Comment:
What are the ramifications of this?  Is this to say that enabling the accelerated DHT client will perform no worse than when it's disabled (other than the fact that more resources are used) since it is purely additive?

Reply:
This is a technicality, we do not trust (/ have not tested) the fullrt (strategy used by the accelerated client) to not cause harm to the overall network health if used at wide scale for routing.

This last sentence explains that you will continue to use use the well tested bucket RT for responding to DHT requests from other nodes. So by you turning on the accelerated DHT client you are not accelerating others DHT lookups faster, however this is at the gain of a reasonable believe that the network topology will not topple over.

Events like https://blog.ipfs.tech/2023-ipfs-unresponsive-nodes/ could very well have been much worst if thoses 50% of nodes didn't just affected the final client doing the lookup, but also the answer you are getting from the 50% of the network that is up. Or they could have not, I don't know we havn't tested that with fullrt.

---

We already running 2 (lan, wan) x 2 (client, server) DHTs by default, all running bucket RT. AcceleratedDHTClient change this to: fullrt client, bucketrt server.

Reply:
Wait this text says the standard client still runs
I don't belive this is true, only standard server no standard server.

Reply:
This is correct.